src/types/device-authentication/index.ts

```diff
@@ -1,3 +1,5 @@
+/* tslint:disable */
```

Comment:
Same thing with disabling TSLint, can you disable the rule(s) which are giving you trouble rather than not linting this file at all?

Reply:
Hi Mathieu, its connected with https://github.com/dvsa/mobile-examiner-alpha/pull/55#discussion_r180983592 (can we continue thread there?)